.gitlab-ci.yml

```diff
@@ -307,52 +303,53 @@ test:helm_chart_upgrade_from_last_tls_to_3_6:
   script:
     - tests/ci-make-deps.sh
     - |
-      echo "INFO - installing mender 3.3"
+      echo "INFO - installing mender tag ${MENDER_PREVIOUS_RELEASE_TAG:-mender-3.4.0} with helm chart version ${MENDER_LTS_HELM_VERSION:-5.0.1}"
       source ./tests/variables.sh
       helm upgrade -i mender \
         -f mender/values.yaml \
         -f tests/keys.yaml \
         -f tests/values-helmci.yaml \
         --wait \
         --timeout=${HELM_UPGRADE_TIMEOUT:-15m} \
-        --set global.image.tag="mender-3.3" \
+        --set global.image.tag="${MENDER_PREVIOUS_RELEASE_TAG:-mender-3.4.0}" \
         --set global.image.username=${REGISTRY_MENDER_IO_USERNAME} \
         --set global.image.password="${REGISTRY_MENDER_IO_PASSWORD}" \
         --set global.s3.AWS_ACCESS_KEY_ID="${MINIO_accessKey}" \
         --set global.s3.AWS_SECRET_ACCESS_KEY="${MINIO_secretKey}" \
-        --namespace mender-upgrade-test-3-3 \
-        mender/mender || exit 3;
+        --namespace ${NAMESPACE} \
+        mender/mender --version ${MENDER_LTS_HELM_VERSION:-5.0.1} || exit 3;
     - bash tests/test-001-wait-for-pods-to-be-ready.sh
     - |
-      echo "INFO - installing this mender"
+      echo "INFO - installing mender tag ${MENDER_CURRENT_LTS_RELEASE_TAG:-mender-3.6.0}"
       helm upgrade -i mender \
         -f mender/values.yaml \
         -f tests/keys.yaml \
         -f tests/values-helmci.yaml \
         --wait \
         --timeout=${HELM_UPGRADE_TIMEOUT:-15m} \
-        --set global.image.tag="mender-3.4" \
+        --set global.image.tag="${MENDER_CURRENT_LTS_RELEASE_TAG:-mender-3.6.0}" \
         --set global.image.username=${REGISTRY_MENDER_IO_USERNAME} \
         --set global.image.password="${REGISTRY_MENDER_IO_PASSWORD}" \
         --set global.s3.AWS_ACCESS_KEY_ID="${MINIO_accessKey}" \
         --set global.s3.AWS_SECRET_ACCESS_KEY="${MINIO_secretKey}" \
-        --namespace mender-upgrade-test-3-3 \
+        --namespace ${NAMESPACE} \
         ./mender || exit 3;
     - helm history mender
     - make test
 
-test:helm_chart_upgrade_from3.4:
+test:helm_chart_upgrade_from_current_lts_to_current_release:
   tags:
     - mender-qa-worker-generic-light
   image: ${PIPELINE_TOOLBOX_IMAGE}
   stage: test
   needs: ["build:setup_eks_cluster"]
+  allow_failure: true
   dependencies:
     - build:setup_eks_cluster
   rules:
     - if: '$CI_COMMIT_BRANCH == "master" || $CI_COMMIT_BRANCH == "master-next" || $RUN_HELM_CHART_INSTALL == "true" || $CI_COMMIT_TAG'
   variables:
-    NAMESPACE: "mender-upgrade-test-3-4"
+    NAMESPACE: "mender-upgrade-test-current-lts"
```

Comment:
```suggestion
    NAMESPACE: "mender-upgrade-test-current-lts"
    TARGET_VERSION: "3.6.0"
```

Reply:
I'll leave this refactoring to @oldgiova